CryptoTechReminderSystem.Test/Gateway/ExcludedSlackUsersExampleResponse.json

```diff
@@ -0,0 +1,131 @@
+      "is_ultra_restricted": true
+    },
+    {
+      "id": "W123AMON",
+      "team_id": "T012AB3C4",
+      "deleted": false,
+      "name": "Monica",
+      "real_name": "Monica Geller-Bing",
+      "tz": "America/Los_Angeles",
+      "tz_label": "Pacific Daylight Time",
+      "tz_offset": -25200,
+      "profile": {
+        "display_name": "Monica",
+        "email": "monica@friends.com"
+      },
+      "is_bot": false,
+      "is_ultra_restricted": true
+    },
+    {
+      "id": "W789ROSS",
+      "team_id": "T012AB3C4",
+      "deleted": true,
+      "name": "Ross",
+      "real_name": "Ross Geller",
+      "tz": "America/Los_Angeles",
+      "tz_label": "Pacific Daylight Time",
+      "tz_offset": -25200,
+      "profile": {
+        "display_name": "Ross",
+        "email": "ross@friends.com"
+      },
+      "is_bot": false,
+      "is_ultra_restricted": false
+    },
+    {
+      "id": "W345RACH",
+      "team_id": "T012AB3C4",
+      "deleted": true,
+      "name": "Rachel",
+      "real_name": "Rachel Green",
+      "tz": "America/Los_Angeles",
+      "tz_label": "Pacific Daylight Time",
+      "tz_offset": -25200,
+      "profile": {
+        "display_name": "Rachel",
+        "email": "rachel@friends.com"
+      },
+      "is_bot": false,
+      "is_ultra_restricted": false
+    },
+    {
+      "id": "W345JOEY",
+      "team_id": "T012AB3C4",
+      "deleted": false,
+      "name": "Joey",
+      "real_name": "Joey Tribbani",
+      "tz": "America/Los_Angeles",
+      "tz_label": "Pacific Daylight Time",
+      "tz_offset": -25200,
+      "profile": {
+        "display_name": "Joey",
+        "email": "joey@friends.com"
+      },
+      "is_bot": false,
+      "is_ultra_restricted": false
+    },
+    {
+      "id": "W5673PHO",
+      "team_id": "T012AB3C4",
+      "deleted": false,
+      "name": "Phoebe",
+      "real_name": "Phoebe Buffay-Hannigan",
+      "tz": "America/Los_Angeles",
+      "tz_label": "Pacific Daylight Time",
+      "tz_offset": -25200,
+      "profile": {
+        "display_name": "Phoebe",
+        "email": "phoebe@friends.com"
+      },
+      "is_bot": false,
+      "is_ultra_restricted": false
+    },
+    {
+      "id": "W5673SLA",
+      "team_id": "T012AB3C4",
+      "deleted": false,
+      "name": "Slackbot",
+      "real_name": "The Real Slim Shady",
+      "tz": "America/Los_Angeles",
+      "tz_label": "Pacific Daylight Time",
+      "tz_offset": -25200,
+      "profile": {
+        "display_name": "Slackbot"
+      },
+      "is_bot": false,
+      "is_ultra_restricted": false
+    },
+    {
+      "id": "W567WOLF",
+      "team_id": "T012AB3C4",
+      "deleted": false,
+      "name": "Wolfbot",
+      "real_name": "WolfBoiiii",
+      "tz": "America/Los_Angeles",
+      "tz_label": "Pacific Daylight Time",
+      "tz_offset": -25200,
+      "profile": {
+        "display_name": "Wolfbot"
+      },
+      "is_bot": true,
+      "is_ultra_restricted": false
+    }
+  ]
+}
```

Comment:
Is it necessary to keep these in separate json files rather than having one?

Reply:
It's for clarity to split the tests so that there are tests where filtering occurs on users - and tests where it doesn't occur (the original tests with count of 6)